Assets/Scripts/Models/Tile.cs

```diff
@@ -410,7 +409,7 @@ public IEnumerable<ContextMenuAction> GetContextMenuActions(ContextMenu contextM
             {
                 Text = "Cancel Job",
                 RequireCharacterSelected = false,
-                Action = (cm, c) => 
+                Action = (cm, c) =>
```

Comment:
We could, but I don't want to expand the scope of this PR too far. It's already big in terms of touched lines.